Binary files kolourpaint-3.5.10.orig/kolourpaint/.Makefile.am.swp and kolourpaint-3.5.10/kolourpaint/.Makefile.am.swp differ
diff -urpN kolourpaint-3.5.10.orig/kolourpaint/Makefile.am kolourpaint-3.5.10/kolourpaint/Makefile.am
--- kolourpaint-3.5.10.orig/kolourpaint/Makefile.am	2005-09-10 01:19:07.000000000 -0700
+++ kolourpaint-3.5.10/kolourpaint/Makefile.am	2012-12-09 10:49:45.120287677 -0800
@@ -45,7 +45,8 @@ kolourpaint_LDADD = $(LIB_KDEPRINT) \
     pixmapfx/libkolourpaintpixmapfx.la \
     tools/libkolourpainttools.la \
     views/libkolourpaintviews.la \
-    widgets/libkolourpaintwidgets.la
+    widgets/libkolourpaintwidgets.la \
+    -lkdefx

 AM_CPPFLAGS = -I$(srcdir)/cursors -I$(srcdir)/interfaces \
     -I$(srcdir)/pixmapfx \
